### PR TITLE
Milestone 5 [Bookstore]: Styling

### LINKED ISSUE #5: [2 Hrs] Styling.
This is the last phase of the Bookstore website project built using React and Redux. Now that all the functionality is done, I will add styling. Here I will use Zeplin as a collaboration tool.

### PR DESCRIPTION
**Pull request by cherelemma**
# Implementation Summary

In this milestone bookstore project, the website is styled following react best practices and language style guides in code.

## General Requirements

✅  No linters error.
✅  Followed the Git-flow.
✅ Followed JS, HTML&CSS best practices
✅  Documented work professionally.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -9,20 +9,49 @@ const Book = ({ book }) => {
...
+        </div>
+      </div>
+
+      <div className="process">
+        <div className="curr-chapter">
+          <p className="completed">Current Chapter</p>
+          <p className="chapter">Chapter  17</p>
+        </div>
+        <button type="button">UPDATE PROCESS</button>
+      </div>
```
Comment (TedLivist) [inline src/components/Book.js]: - [ ] Kindly note that you are required to replicate the styling exactly as it is on the design. Kindly use the correct fonts and sizes for the texts as we have in the design.

- [x] Kindly take note of the background color. The original design has a somewhat lighter grey color but yours is darker. Kindly fix that as well.

  Your Implementation | Zeplin Design
  :---:|:---:
  wiwow | euwo

File: src/components/NewBook.js [hunk 2]
```diff
@@ -27,16 +27,16 @@ const AddBookForm = () => {
...
     <div className="form-wrapper">
-      <h1>ADD NEW BOOK</h1>
+      <h3>ADD NEW BOOK</h3>
       <form className="book-form" onSubmit={handleSubmit}>
         <input
-          className="book-input"
+          className="title-input"
           type="text"
           name="title"
           value={userInput.title}
```
Comment (TedLivist) [inline src/components/NewBook.js:42]: - [ ] Kindly ensure you style the form component as well to fit the design - noting the fonts and sizes of the text.

  Your Implementation | Zeplin Design
  :---:|:---:
  wqp | ewpw

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @cherelemma,

Good job so far!
✔️ Good documentation
✔️ Nice styling of navbar
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — APPROVED** [review]
Hi @cherelemma, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**